### PR TITLE
Add Bionic end-of-support post

### PR DESCRIPTION
**Pull request by sophiewigmore**
## Summary

blog post about the end of support for bionic in the project

## Checklist

* [X] I have viewed, signed, and submitted the Contributor License Agreement.
* [ ] I have linked issue(s) that this PR should close using keywords or the Github UI (See [docs](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue))
* [ ] I have added an integration test, if necessary.
* [X] I have reviewed the [styleguide](https://github.com/paketo-buildpacks/community/blob/main/STYLEGUIDE.md) for guidance on my code quality.
* [X] I'm happy with the commit history on this PR (I have rebased/squashed as needed).

### CODE REVIEW

**Review by robdimsdale — DISMISSED** [review]
Great - thank you for putting this together!

File: content/posts/0022-bionic-eos.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+As of May 31, 2023, Ubuntu 18.04 (Bionic Beaver) is [out of
+support](https://ubuntu.com/blog/18-04-end-of-standard-support) for open-source
+customers.
+
+As a result, Paketo will stop supporting Bionic in the project. If you
+currently depend on our Bionic-based offerings, this blog post will contain
+all of the relevant information about what this means and how to migrate.
+
+## Deprecation Details
```
Comment (arjun024) [inline content/posts/0022-bionic-eos.md:16]: Nit: s/Deprecation/End of Support/? (no strong opinion)

**Review by arjun024 — DISMISSED** [review]

File: content/posts/0022-bionic-eos.md (hunk 1 above)
Comment (loewenstein) [reply content/posts/0022-bionic-eos.md:16]: Given this times timing the thing happening is an EOL. However, if we want to set precedent for the future, we should announce the deprecation of Jammy way before we stop publishing updates...

So, I agree that both would work here.

File: content/posts/0022-bionic-eos.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+The removal of Bionic support in Paketo was announced and ratified in the
+[Bionic End of Support RFC](https://github.com/paketo-buildpacks/rfcs/blob/main/text/0057-bionic-eos.md).
+The following stacks and builders will no longer be supported, meaning
+  - (1) we will no longer publish updates (meaning no CVE fixes),
+  - (2) the related repositories have been archived, and
+  - (3) the images will be removed from Docker and GCR in accordance with our [image retention policy](https://github.com/paketo-buildpacks/rfcs/blob/main/text/0046-image-retention-policy.md)
```
Comment (dmikusa) [inline content/posts/0022-bionic-eos.md:23]:
```suggestion
  - (3) the images will be removed from Docker and GCR in accordance with our [image retention policy](https://github.com/paketo-buildpacks/rfcs/blob/main/text/0046-image-retention-policy.md), which states we'll retain them for at least two years.
```

Not necessary to make them click through just to find the important part.

File: content/posts/0022-bionic-eos.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+We strongly recommend migrating your workloads from Bionic to our Ubuntu 22.04
+(Jammy Jellyfish) based offerings. Each Bionic stack and builder listed above
+has a more or less 1:1 Jammy equivalent available in the project.
+
+If you are using the `pack` CLI version v0.29.0 or prior for your builds, you
+can set the default builder for your builds to Jammy via the `pack config default-builder` command:
+```
+pack config default-builder paketobuildpacks/builder-jammy-base
+```
+In future versions of `pack`, Jammy will be the default.
```
Comment (dmikusa) [inline content/posts/0022-bionic-eos.md:49]: I don't think this is true. Pack doesn't have a default builder. You always need to set a default builder, if you want a default to be applied.

What will be updated in the next pack release is the output from `pack builder suggest`. Presently, it still suggests the EOL builders. We are waiting on upstream to release pack so it suggests the correct things.

Comment (dmikusa) [reply content/posts/0022-bionic-eos.md:49]:
```suggestion
You may notice that current versions of `pack` still suggest using the Bionic builder if you run `pack builder suggest`. This is out-of-date and Paketo is waiting on a new release of `pack` which will correctly suggest the Jammy builders. Please ignore the output of `pack builder suggest` for now and use the builder listed above.
```

File: content/posts/0022-bionic-eos.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+
```
Comment (dmikusa) [inline content/posts/0022-bionic-eos.md:50]:
```suggestion

If you are using Spring Boot build tools, you will need to set the builder in your `pom.xml` if you're using Maven or in your `build.gradle` if you're using Gradle. For Maven, instructions [see here](https://docs.spring.io/spring-boot/docs/3.1.2/maven-plugin/reference/htmlsingle/#build-image.examples.custom-image-builder). For Gradle instructions, [see here](https://docs.spring.io/spring-boot/docs/3.1.2/gradle-plugin/reference/htmlsingle/#build-image.examples.custom-image-builder). Use one of the builder's listed above, like `paketobuildpacks/builder-jammy-base`.
```

File: content/posts/0022-bionic-eos.md (hunk 3 above)
Comment (dmikusa) [inline content/posts/0022-bionic-eos.md:49]:
```suggestion

Here is a complete list of the new builders:

- `paketobuildpacks/builder-jammy-full`
- `paketobuildpacks/builder-jammy-base`
- `paketobuildpacks/builder-jammy-tiny`
- `paketobuildpacks/builder-jammy-buildpackless-full`
- `paketobuildpacks/builder-jammy-buildpackless-base`
- `paketobuildpacks/builder-jammy-buildpackless-tiny`
- `paketobuildpacks/builder-jammy-buildpackless-static`

In future versions of `pack`, Jammy will be the default.
```

**Review by dmikusa — CHANGES_REQUESTED** [review]
Added some comments/suggestions.

File: content/posts/0022-bionic-eos.md (hunk 1 above)
Comment (sophiewigmore) [reply content/posts/0022-bionic-eos.md:16]: Ill change it to End of Support!

File: content/posts/0022-bionic-eos.md [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+(Jammy Jellyfish) based offerings. Each Bionic stack and builder listed above
+has a more or less 1:1 Jammy equivalent available in the project.
+
+If you are using the `pack` CLI version v0.29.0 or prior for your builds, you
+can set the default builder for your builds to Jammy via the `pack config default-builder` command:
+```
+pack config default-builder paketobuildpacks/builder-jammy-base
+```
+You may notice that current versions of `pack` still suggest using the Bionic builder if you run `pack builder suggest`. This is out-of-date and Paketo is waiting on a new release of `pack` which will correctly suggest the Jammy builders. Please ignore the output of `pack builder suggest` for now and use the builder listed above.
+
```
Comment (sophiewigmore) [inline content/posts/0022-bionic-eos.md:50]:
```suggestion
Here is a complete list of the new builders:
- `paketobuildpacks/builder-jammy-full`
- `paketobuildpacks/builder-jammy-base`
- `paketobuildpacks/builder-jammy-tiny`
- `paketobuildpacks/builder-jammy-buildpackless-full`
- `paketobuildpacks/builder-jammy-buildpackless-base`
- `paketobuildpacks/builder-jammy-buildpackless-tiny`
- `paketobuildpacks/builder-jammy-buildpackless-static`
```

**Review by dmikusa — APPROVED** [review]
Awesome!

**Review by ForestEckhardt — APPROVED** [review]

**Pull request merged**